prdoc/pr_7784.prdoc

```diff
@@ -0,0 +1,11 @@
+title: '[pallet-revive] block.timestamps should return seconds'
+doc:
+- audience: Runtime Dev
+  description: |-
+    In solidity `block.timestamp` should be expressed in seconds
+    see https://docs.soliditylang.org/en/latest/units-and-global-variables.html#block-and-transaction-properties
+crates:
+- name: pallet-revive
+  bump: minor
+- name: pallet-revive-uapi
+  bump: minor
```

Comment:
Patch is fine here. Will not break compilation for anybody depending on `pallet_revive`. This is important for backport.

```suggestion
- name: pallet-revive
  bump: patch
- name: pallet-revive-uapi
  bump: patch
```